README.md

```diff
@@ -230,52 +287,69 @@ When uploading WAL archives to S3, the user should pass in the absolute path to
 wal-g wal-push /path/to/archive
 ```
 
-* ``backup-list``
+### MySQL
 
-Lists names and creation time of available backups.
+* ``stream-fetch``
```

Comment:
stream-fetch is not mysql-specific tool. why it's in mysql section ?

Reply:
Yep, but it still needs refactoring for making it common. Is it okay if we place it here right now, and when I make it common move it to a common part of README?

Reply:
ok